interface/batchcom/batchPhoneList.php

```diff
@@ -6,42 +6,45 @@
 // create a list for phone calls
 // menu for fields could be added in the future
```

Comment:
Lets convert the above lines of code into a nice compact header. See here for format to use (just place 1 line for the author and 1 line for the copyright with your information):
http://www.open-emr.org/wiki/index.php/How_to_Document_Your_Code_Properly#File-Level_DocBlock